server/requirements.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements-prod.txt
+coverage
+nose
+mock
+requests-mock
```

Comment:
Oh this is nifty, I didn't know that you could make install stuff from a *different* requirements file!